slither/core/declarations/function.py

```diff
@@ -306,7 +306,11 @@ def can_reenter(self, callstack=None) -> bool:
         if self._can_reenter is None:
             self._can_reenter = False
             for ir in self.all_slithir_operations():
-                if isinstance(ir, Call) and ir.can_reenter(callstack):
+                if (
+                    isinstance(ir, Call)
+                    and ir.can_reenter(callstack)
+                    and ir.function_name != "staticcall"
```

Comment:
`ir.function_name` is not a member of `Call`, so this can crash. I think we should add this check in `LowLevelCall.can_reenter`: https://github.com/crytic/slither/blob/5021f3e7a04e46dd32655f8d168b43a8bb000770/slither/slithir/operations/low_level_call.py#L59-L64